pkg/kubectl/cmd/cmd.go

```diff
@@ -219,69 +219,64 @@ func NewKubectlCommand(f cmdutil.Factory, in io.Reader, out, err io.Writer) *cob
 
 	groups := templates.CommandGroups{
 		{
-			Message: "Basic Commands (Beginner):",
+			Message: "Application Management (Using Command Args / Flags)",
 			Commands: []*cobra.Command{
-				NewCmdCreate(f, out),
+				NewCmdAnnotate(f, out),
+				NewCmdAutoscale(f, out),
 				NewCmdExposeService(f, out),
+				NewCmdLabel(f, out),
+				rollout.NewCmdRollout(f, out),
 				NewCmdRun(f, in, out, err),
+				NewCmdScale(f, out),
 				set.NewCmdSet(f, out, err),
 			},
 		},
 		{
-			Message: "Basic Commands (Intermediate):",
+			Message: "Application Management (Either Yaml Or Command Args)",
```

Comment:
That's still really verbose - as a user, I'm not sure that helps me.  The concept we're trying to convey is "Use these commands to manage your applications directly" which is slightly different than "Use these commands to treat your applications as config" or "manage applications using config".